src/Illuminate/Support/helpers.php

```diff
@@ -8,6 +8,24 @@
 use Illuminate\Contracts\Support\Htmlable;
 use Illuminate\Support\HigherOrderTapProxy;
 
+if (! function_exists('any_empty')) {
+    /**
+     * Check if any of the given variables is empty.
+     *
+     * @return string
+     */
+    function any_empty()
+    {
+        foreach (func_get_args() as $arg) {
+            if (empty($arg) === true) {
```

Comment:
you don't have to do the equality check since `empty()` already returns a boolean